jax/_src/extend/ffi.py

```diff
@@ -14,7 +14,7 @@
 
 from __future__ import annotations
 
-from collections.abc import Iterable, Mapping, Sequence
+from collections.abc import Iterable, Literal, Mapping, Sequence
```

Comment:
It looks like this is maybe the problem:

```suggestion
from collections.abc import Iterable, Mapping, Sequence
```

Reply:
Indeed, hadn't pushed the cleanup. Thanks!